go.mod

```diff
@@ -6,6 +6,7 @@ require (
 	github.com/anchore/stereoscope v0.0.0-20220708133445-777471f38c5b
 	github.com/anchore/syft v0.53.0
 	github.com/davecgh/go-spew v1.1.1
+	github.com/digitorus/timestamp v0.0.0-20220704143351-8225fba02d52
```

Comment:
Eventually, I'd like this to be in tree or a fork under our control

Reply:
Maintaining a fork of every lib we use is a maintenance nightmare

Reply:
As far as what data actually is that is an implementation detail of the timestamper being used. 

In the case of TSP it’s an ASN1 encoded Timestamp Request object.

Reply:
We need to be judicial about what we fork and why. 

Forking repos incurs maintenance costs and burdens.

Reply:
I looked into it a bit more; the repo is maintained by a real company, https://www.digitorus.com/, based in the Netherlands that specializes in PKI.  I don't see any immediate concerns.

Initially, seeing a small repo with only two maintainers on a critical code path gives me pause.